invenio_communities/views/ui.py

```diff
@@ -177,6 +176,10 @@ def new():
             for at least one week. If you want to speed up the process, you \
```

Comment:
Instead of `\`, it's better to use string literal lines in parentheses, just to be sure about the whitespace